cmake/OTelMakeDeb.cmake

```diff
@@ -0,0 +1,27 @@
+set(CMAKE_PROJECT_NAME "opentelemetry-cpp")
+set(CPACK_SYSTEM_NAME "${OS_RELEASE_ID}-${OS_RELEASE_VERSION_ID}")
+set(CPACK_PACKAGING_INSTALL_PREFIX "/usr/local")
+set(CPACK_GENERATOR "DEB")
+
+set(MAJOR_VERSION "1")
+set(MINOR_VERSION "6")
+set(PATCH_VERSION "1")
+
+set(CPACK_PACKAGE_DESCRIPTION "OpenTelemetry C++ for Linux")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "OpenTelemetry C++ for Linux - C++ Implementation of OpenTelemetry Specification")
+set(CPACK_PACKAGE_VENDOR "OpenTelemetry")
+set(CPACK_PACKAGE_CONTACT "OpenTelemetry-cpp")
+set(CPACK_PACKAGE_VERSION_MAJOR "${MAJOR_VERSION}")
```

Comment:
Why not use `${PROJECT_VERSION_MAJOR}` , `${PROJECT_VERSION_MINOR}` and `${PROJECT_VERSION_PATCH}` ?

Reply:
Using `project` will will make it clearer.

```cmake
project(opentelemetry-cpp VERSION 1.8.0)
```

> **CPACK_PACKAGE_VERSION_MAJOR**
Package major version. This variable will always be set, but its default value depends on whether or not version details were given to the [project()](https://cmake.org/cmake/help/latest/command/project.html#command:project) command in the top level CMakeLists.txt file.

Reply:
Have modified the code to use `OPENTELEMETRY_VERSION` instead.